i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-kubernetes-api.md

```diff
@@ -1089,3 +1137,289 @@ utiliser l'option `--debug` pour obtenir l'erreur exacte.
+
+#### Options des custom modes
+
+Les options spécifiques aux **custom modes** sont listées ci-dessous :
+
+<Tabs groupId="sync">
+<TabItem value="api" label="api">
+
+| Option            | Description                                                                                                                                                                                                                                 |
+|:------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --hostname        | Kubernetes API hostname.                                                                                                                                                                                                                    |
+| --port            | API port (default: 443)                                                                                                                                                                                                                     |
+| --proto           | Specify https if needed (default: 'https')                                                                                                                                                                                                  |
+| --timeout         | Set HTTP timeout                                                                                                                                                                                                                            |
+| --limit           | Number of responses to return for each list calls. See https://kubernetes.io/docs/reference/kubernetes-api/common-param eters/common-parameters/#limit                                                                                      |
+| --namespace       | Set namespace to get information.                                                                                                                                                                                                           |
+| --legacy-api-beta | If this option is set the legacy API path are set for this API calls: kubernetes\_list\_cronjobs will use this path: /apis/batch/v1beta1/namespaces/ kubernetes\_list\_ingresses will use this path: /apis/extensions/v1beta1/namespaces/  This ways are no longer served since K8S 1.22 see https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122   |
```

Comment:
```suggestion
| --legacy-api-beta | If this option is set the legacy API path are set for this API calls: kubernetes\_list\_cronjobs will use this path: /apis/batch/v1beta1/namespaces/ and kubernetes\_list\_ingresses will use this path: /apis/extensions/v1beta1/namespaces/ . This ways are no longer served since K8S 1.22 see https://kubernetes.io/docs/reference/using-api/deprecation-guide/#ingress-v122   |
```